### DISCUSSION TITLE
How do I use the modules?

### DISCUSSION THREAD

**Discussion by vdawg-git** [abenz1267/walker#114] [Q&A] [answered]
I launch walker as a service via `exec = walker --gapplication-service`.

But how do I access modules like `clipboard`, `emojies` etc.?

**Comment by abenz1267** [comment, marked as answer]
You can access them by using the switcher module, which is prefixed with`/`. So `/clipboard`. Or you can do `walker -m clipboard`.

**Reply by vdawg-git** [reply]
Thank you :)
